### ISSUE TITLE
can not test 'have_record_set()'

### ISSUE THREAD

**Issue by jedipunkz** [k1LoW/awspec#97]
hi

we have a record named 'bar.foo.internal.' in a hosted zone named 'foo.internal.'
and its record have a 'A record' with value '10.220.1.240' and ttl 300.

then i wrote test_spec.rb such as

```
describe route53_hosted_zone('foo.internal.') do
    it { should exist }
    it { should have_record_set('bar.foo.internal.').a('10.220.1.240').ttl(300) }
end
```

and i got these errors.

```
route53_hosted_zone 'foo.internal.'
  should exist
  should have record set "bar.foo.internal." (FAILED - 1)

Failures:

  1) route53_hosted_zone 'foo.internal.' should have record set "bar.foo.internal."
     Failure/Error: it { should have_record_set('bar.foo.internal.').a('10.220.1.240').ttl(300) }
       expected route53_hosted_zone 'bar.internal.' to have record set "bar.foo.internal."
     # ./spec/test_spec.rb:18:in `block (2 levels) in <top (required)>'

Finished in 0.02945 seconds (files took 2.62 seconds to load)
2 examples, 1 failure

Failed examples:

rspec ./spec/test_spec.rb:18 # route53_hosted_zone 'foo.internal.' should have record set "bar.foo.internal."
```

do i mistake to write test_spec.rb ?

**Comment by k1LoW** [comment]
try this

## STEP1: Get `foo.internal.` hoste-zone-id

```
$ aws route53 list-hosted-zones
{
    "HostedZones": [
        {
            "ResourceRecordSetCount": 2,
            "CallerReference": "XXXXX-XXXXX-XXXX-XXXX-XXX",
            "Config": {
                "PrivateZone": false
            },
            "Id": "/hostedzone/ABCDEFG",
            "Name": "foo.internal."
        }
    ]
}
```

## STEP2: Check `foo.internal.` record sets

```
$ aws route53 list-resource-record-sets --hoste-zone-id ABCDEFG
```

And, please share output about `bar.foo.internal.` record.

**Comment by k1LoW** [comment]
or try this (more easy)

```
$ awspec generate route53_hosted_zone foo.internal.
```

**Comment by jedipunkz** [comment]
i checked record by aws cli.

```
            "ResourceRecords": [
                {
                    "Value": "10.220.1.240"
                }
            ],
            "Type": "A",
            "Name": "bar.foo.internal.",
            "TTL": 300
        },
```

I noticed that 'awspec generate' can not output 100 more records. so i have 178 record in foo.internal. hosted zone.

```
$ awspec generate route53_hosted_zone foo.internal. | grep bar
$ awspec generate route53_hosted_zone foo.internal. | grep 'have_record_set' | wc -l
     100
```

can not awspec list record_sets more 100 ?

**Comment by k1LoW** [comment]
> I noticed that 'awspec generate' can not output 100 more records.

Oops. it`s awspec (aws-sdk call) bug. :scream_cat:

Thank you for your report! I try fix this.

**Comment by k1LoW** [comment]
Fixed as #98 (v0.25.2).

Please try.

**Comment by jedipunkz** [comment]
perfect! thanks!
